include/bluetooth/mesh/sensor.rst

```diff
@@ -153,12 +153,7 @@ Typically, settings should only be meta data related to the sensor data type, bu
 Available sensor types
 **********************
 
-All available sensor types are collected in a single module:
-
-.. toctree::
-   :maxdepth: 1
-
-   sensor_types.rst
```

Comment:
This file is no longer part of the toctree now. Can it be re-added somewhere, like here or as a last resort, in mesh.rst?

Reply:
It's already added to mesh.rst :)